### PR TITLE
Restructure examples based on feature-group ids

### PR DESCRIPTION
**Pull request by lootag**

### CODE REVIEW

File: go/alerts-client.go [hunk 1]
```diff
@@ -408,12 +408,12 @@ const (
 
 // Alert API RPC names.
 const (
```
Comment (celaus) [inline go/alerts-client.go:410]: this will break several things

File: go/alerts-client.go [hunk 2]
```diff
@@ -489,16 +489,22 @@ const (
...
+
+	// AlertsOpOr is an and operation for alerts.
+	AlertsOpOr = alerts.AlertsOp_ALERTS_OP_OR
 )
 
 // NextOp is a next operation setting for alerts.
 type NextOp = alerts.NextOp
 
 const (
+	// NextOpAndOrUnspecified is an and/or operation for alerts.
```
Comment (celaus) [inline go/alerts-client.go:503]: better to just comment on the block itself

File: go/apikeys-client.go [hunk 3]
```diff
@@ -58,10 +58,10 @@ type OwnerOrganisationID = apikeys.Owner_OrganisationId
 
 // RPC method names.
 const (
-	CreateAPIKeyRpc = apikeys.ApiKeysService_CreateApiKey_FullMethodName
```
Comment (celaus) [inline go/apikeys-client.go:61]: Same here, this will break a bunch of things in the TF provider

File: go/actions-client.go [hunk 4]
```diff
@@ -46,21 +46,38 @@ type Action = actions.Action
...
+
+	// SourceTypeLog is a webhook source type.
+	SourceTypeLog = actions.SourceType_SOURCE_TYPE_LOG
+
+	// SourceTypeDataMap is a data map source type.
+	SourceTypeDataMap = actions.SourceType_SOURCE_TYPE_DATA_MAP
 )
 
 const (
-	GetActionRpc                 = actions.ActionsService_GetAction_FullMethodName
```
Comment (celaus) [inline go/actions-client.go:57]: Breaking changes, these. also isn't a block comment suffient?

File: go/scripts/generate_go_protos.sh [hunk 5]
```diff
@@ -37,6 +37,11 @@ do
                 args+="--go_opt=M${proto_file##*$proto_dir/}=${mod_name}/${go_out_dir}/coralogixapis/dashboards/v1 "
                 args+="--go-grpc_opt=M${proto_file##*$proto_dir/}=${mod_name}/${go_out_dir}/coralogixapis/dashboards/v1 "
                 ;;
+            # The archive v2 protos contain circular dependencies, so we need to make sure that all dashboard files end up in the same package
```
Comment (celaus) [inline go/scripts/generate_go_protos.sh:40]: remove this duplicate

**Review by celaus — APPROVED** [review]

File: go/scripts/generate_go_protos.sh (hunk 5 above)
Comment (lootag) [reply go/scripts/generate_go_protos.sh:40]: Done :-)

File: go/actions-client.go (hunk 4 above)
Comment (lootag) [reply go/actions-client.go:57]: Nope, every public constant needs a comment

**Pull request merged**